### DISCUSSION TITLE
How to export podman image distribution on Mac

### DISCUSSION THREAD

**Discussion by shrutscode** [containers/podman#22387] [Q&A] [answered]
I am trying to create a custom distribution to initialise a podman machine using the Fedora distribution.
(https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/39.20240104.2.0/aarch64/fedora-coreos-39.20240104.2.0-qemu.aarch64.qcow2.xz)
podman machine init --image-path (path-to-above-image-download)
Post this I have to add some customisation to the image to make it work within my organisation, so basically adding certificates etc.
After customisation, I want to export the custom image to my local and then use this new distribution as the base.
How can I do export this image in Mac? (in Windows I was able to use wsl export command after customising the amd64 distribution)
I am an absolute beginner, I hope I have articulated the issue correctly.
Thanks.

**Comment by baude** [comment, marked as answer]
if you are an absolute beginner, this is not going to be trivial.  You can use the [podman-machine-os](https://github.com/containers/podman-machine-os) repo as a guide.  The distributions, except for WSL, are generated using a Containerfile and based on Fedora CoreOS.  As such, they also need ignition (like cloud-init) and other customizations needed.  It is probably too much to explain here.

One shortcut you could make is to boot a machine.  The create a Containerfile which has a `FROM quay.io/podman/machine-os:TAG` to start.  Then using the Containerfile fro the podman-machine-os repo as an example, you can make modifications.  Once that image is built, you can use `podman machine os apply` to lay it in.

Good luck!

**Comment by shrutscode** [comment]
Thanks for your response. Yes this is the right direction. I will take it up with my team.
